src/components/ReportActionItem/MoneyRequestView.tsx

```diff
@@ -141,7 +143,8 @@ const perDiemPoliciesSelector = (policies: OnyxCollection<OnyxTypes.Policy>) =>
 
 function MoneyRequestView({
     allReports,
-    report,
+    report: transactionThreadReport,
+    parentReportProp,
```

Comment:
Also, https://expensify.slack.com/archives/C05LX9D6E07/p1765986258747939 suggests that just passing the `parentReportID` then subscribing only to the data from that parent report that's actually needed by this component would be a better practice.

Reply:
I think this would require investigating all the helper functions that take it as param and see what data they use from it. I think it might be out of scope here but can be a follow up to clean up the code of this component. I'm also noticing the transaction thread report is merely used for route generations for editing fields